roles/browsertrix/templates/crawl.sh.j2

```diff
@@ -4,10 +4,11 @@
 # docker commands need to be run as root unless otherwise configured
 sudo docker run -v {{ browsertrix_crawl_config_dest }}:/app/crawl-config.yaml \
     -v {{ browsertrix_crawl_dir }}:/crawls/ webrecorder/browsertrix-crawler crawl \
-    {% if browsertrix_custom_driver is defined %}-v {{ browsertrix_custom_driver_dest }}:/app/defaultDriver.js \{% endif %}
+    {% if browsertrix_custom_driver is defined %}-v {{ browsertrix_custom_driver_dest }}:/app/custom-driver.js \ {% endif %}
```

Comment:
@rlskoeser I didn't know what was causing the lack of a newline. Sublime Text highlighting suggested that we might have been escaping one of the curly brackets? Let me know if you think it's something else